api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -559,7 +559,7 @@ private ValuesLinkingPair getValueSelectsAndJoins(List<DomainValuePair> domainVa
             .collect(Collectors.toList()));
 
     final String domainName = domainMaybe.get().toString();
-    final String domainTitleCase = toTitleCase(domainName);
+    final String domainTitleCase = capitalizeFirstCharacterOnly(domainName);
```

Comment:
also rename `domainTitleCase`

Reply:
Since the domain is always a single word, technically it's still title case. The reason for the rename was if this method were moved or exposed more generally.

Still, `domainFirstCharacterCapitalized` is fine too.

Reply:
OK then either is fine

Reply:
done